csharp/sources/Valkey.Glide.InterOp/Native/Imports.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using Valkey.Glide.InterOp.Native.Routing;
+
+namespace Valkey.Glide.InterOp.Native;
+
+[EditorBrowsable(EditorBrowsableState.Advanced)]
+[SuppressMessage("ReSharper", "InconsistentNaming")]
+[SuppressMessage("ReSharper", "BuiltInTypeReferenceStyle")]
+public static class Imports
+{
+    /// <summary>
+    /// Special method to free the returned <see cref="Value"/>s.
+    /// MUST be used!
+    /// </summary>
+    /// <remarks>
+    /// Only ever call this on the root-Value instance!
+    /// </remarks>
+    [DllImport(
+        "glide_rs",
+        EntryPoint = "csharp_free_value",
+        CharSet = CharSet.Unicode,
+        CallingConvention = CallingConvention.Cdecl,
+        SetLastError = false
+    )]
+    public static extern void free_value([In] Value value);
+
+    /// <summary>
+    /// Special method to free the returned strings.
+    /// MUST be used instead of calling c-free!
+    /// </summary>
+    /// <remarks>
+    /// If you got a string via <see cref="Value"/>, free the root-Value instance instead!
+    /// </remarks>
+    [DllImport(
+        "glide_rs",
+        EntryPoint = "csharp_free_string",
+        CharSet = CharSet.Unicode,
+        CallingConvention = CallingConvention.Cdecl,
+        SetLastError = false
+    )]
+    public static extern unsafe void free_string([In] byte* handle);
```

Comment:
```suggestion
    public static extern void free_string([In] IntPtr handle);
```
Ideally I'd like to avoid having `unsafe` in C# code at all

Reply:
We can't entirely avoid unsafe in all cases when dealing with low-level manipulation. I can change this method, but doing so would technically make it incorrect within the context of the syntax.